pkg/cmd/projectCreate_test.go

```diff
@@ -44,10 +44,11 @@ func TestCanCreateProjectGoldenPathNoOBR(t *testing.T) {
 	maven := true
 	gradle := false
 	isDevelopment := false
+	packageName := "my.test.pkg"
```

Comment:
just out of interest, how come there are already tests here?

Reply:
discussed.